app/assets/javascripts/client/views/assessments/shared_report.html

```diff
@@ -0,0 +1,151 @@
+<div class='report'>
```

Comment:
There are about three differences between this and the existing report.html.  For now this approach will likely do, but we should consider ways to reuse our view components, since we've got the copy between the two duplicated.